math/mathmore/src/GSLMCIntegrator.cxx

```diff
@@ -466,8 +471,8 @@ const char * GSLMCIntegrator::GetTypeName() const {
 }
 
 ROOT::Math::IntegratorMultiDimOptions  GSLMCIntegrator::Options() const {
-   IOptions * extraOpts = ExtraOptions();
-   ROOT::Math::IntegratorMultiDimOptions opt(extraOpts);
+   auto extraOpts = ExtraOptions();
+   ROOT::Math::IntegratorMultiDimOptions opt(extraOpts.release());
```

Comment:
```suggestion
   ROOT::Math::IntegratorMultiDimOptions opt(ExtraOptions().release());
```
Can be shortened (which also makes clear that the output of `ExtraOptions(I)` is not used anywhere else)